functorch/csrc/BatchRulesStopDecomposition.cpp

```diff
@@ -438,12 +425,11 @@ TORCH_LIBRARY_IMPL(aten, FT_BATCHED_KEY, m) {
   STOP_DECOMPOSE(mode.dimname);
   STOP_DECOMPOSE(mode.dimname_out);
   STOP_DECOMPOSE(moveaxis.int);
```

Comment:
This is also an alias, right?

Reply:
Yes, they are. I just removed those that are tested. Others does not fail and thus are not tested.

Reply:
Interesting. So a lot of the `foo.Scalar` overloads cannot be called directly from Python, but can be hit from C++ (yes, it's weird). We should still remove the stop_decompose because we're pretty sure that they work.